packages/jira-adapter/src/filters/fields/context_options.ts

```diff
@@ -186,7 +199,8 @@ export const setContextOptions = async (
     option => option.optionId !== undefined || option.parentValue === undefined
   )
 
-  const fieldId = getParents(getChangeData(contextChange))[0].value.value.id
+  const fieldId = (await getParents(getChangeData(contextChange))[0]
+    .getResolvedValue(elementsSource)).value.id
```

Comment:
readability nit: it's hard to understand where the async part ends, maybe 
```suggestion
  const fieldId = (
    await getParents(getChangeData(contextChange))[0].getResolvedValue(elementsSource)
  ).value.id
```